### DISCUSSION TITLE
DepthAI Demo v3.0.2 - OAK-D-Lite support

### DISCUSSION THREAD

**Discussion by VanDavv** [luxonis/depthai#537] [Releases]
With this release, OAK-D-Lite support is added. The demo script is able to recognize the device as any other regular device, no extra steps required

<hr /><em>This discussion was created from the release <a href='https://github.com/luxonis/depthai/releases/tag/v3.0.2'>DepthAI Demo v3.0.2 - OAK-D-Lite support</a>.</em>

**Comment by jimmychou9523** [comment]
How can I verify the camera is in USB 10Gbps mode?
